.github/workflows/npm-release.yaml

```
@@ -0,0 +1,126 @@
+name: NPM Packages Release
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)